pkg/apis/extensions/validation/validation.go

```diff
@@ -915,6 +917,25 @@ func validatePSPCapsAgainstDrops(requiredDrops []api.Capability, capsToCheck []a
 	return allErrs
 }
 
+// validateAllowedHostPaths makes sure all allowed host paths follow:
+// 1. does not have any element which is ".."
+func validateAllowedHostPaths(fldPath *field.Path, allowedHostPaths []string) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	for _, targetPath := range allowedHostPaths {
+		// This assumes the kubelet performs the same check to ensure there are no host differences
+		parts := strings.Split(targetPath, string(os.PathSeparator))
```

Comment:
Kubelet doesn't do anything with PSP objects, so remove this comment.

I'm ok adding this check, but I'm actually less concerned with preventing backsteps in the policy (if they explicitly want to allow /foo/../bar, that may be nonsensical or not actually work, but it's not really a security issue)